src/main/cpp/src/hive_hash.cu

```diff
@@ -210,11 +210,227 @@ class hive_device_row_hasher {
       return this->hash_functor.template operator()<T>(col, row_index);
     }
 
+    /**
+     * @brief A structure to keep track of the computation for nested types.
+     */
+    struct col_stack_frame {
+     private:
+      cudf::column_device_view _column;  // the column has only one row
```

Comment:
Again, this is still incorrect. The column has either:
 * One row, if this is the root column or nested under children of a structs column, or
 * Many rows which are the elements of a list if this is nested under child of a lists column.

Reply:
```suggestion
      // The column has either one row (if root column or nested under structs column),
      // or has rows that are elements of a list (if it is child of a lists column).
      cudf::column_device_view _column;
```

Reply:
This column really only has one row. (

Reply:
Ah I see, sorry about that: I missed one place in the code.